### ISSUE TITLE
Accessibility Violation: Text constrast

### ISSUE THREAD

**Issue by mariannebravo** [carbon-design-system/carbon-components-vue#1406]
## Detailed description

> Describe in detail the issue you're having.

 I'm using the Carbon React DatePicker using `Date.Now()` to prevent the user pick a past date (as you can see in the print below). The issue is related to the disabled options when the date is disabled, the accessibility checker is pointing a violation about text contrast, and i tried to access the CSS class using the `:global` variable from Sass, to change the disabled day color and also to try to change the background color,  and i could not solve this problem.
![image](https://user-images.githubusercontent.com/59119687/212923296-6f328709-36e1-473f-8e88-7ebb8e7cb80b.png)

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug.

> Is this issue related to a specific component?

Date Picker

> What did you expect to happen? What happened instead? What would you like to see changed?

Expected to not show the violation. I'd like to change the color for disabled dates or change the background color for better contrast.

> What browser are you working in?

Chrome, but I also tested on Firefox and Sarafi. Occurs the same violation error.

> What version of the Carbon Design System are you using?

```
"carbon-components": "^10.58.3"
"carbon-components-react": "^7.59.4",
```

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/59119687/212922460-f950d469-c1ce-48a2-94f1-788317bc9160.png)

Code:
```
<DatePicker
              id="date-picker"
              datePickerType="single"
              minDate={Date.now()}
              onChange={(event) => updateDate(event)}
              dateFormat="Y-m-d"
              light
            >
              <DatePickerInput
                id="date-picker-single"
                labelText="Choose a date"
                placeholder={convertDate(currentDate)}
              />
            </DatePicker>
```
